### DISCUSSION TITLE
Connecting to Replica

### DISCUSSION THREAD

**Discussion by mosnicholas** [supabase/supabase#16301] [Questions]
I've setup a database replica to be able to query production data in retool without affecting our prod DB loads. The replica was generated as:

```
CREATE PUBLICATION analytics_readonly FOR TABLE ... with (publish = 'insert')
```

How could I query the replica in Retool vs. the original database?

**Comment by mosnicholas** [comment]
has anyone done this before? :D

**Comment by BoscoDomingo** [comment]
I'm in a similar position except I want to export that data to BigQuery via a CDC tool. I have set up everything correctly and yet all of them fail when testing against Supabase for whatever reason. Would love to know why
